src/main/java/org/opensearch/ad/util/Bwc.java

```diff
@@ -18,13 +18,16 @@
  *
  */
 public class Bwc {
+    public static boolean DISABLE_BWC = true;
```

Comment:
Why does this need to be a variable?

Reply:
I am using it in testing https://github.com/opensearch-project/anomaly-detection/blob/df40978d82c0ea652b05d3c9869e71b296789a20/src/test/java/org/opensearch/BwcTests.java#L102

Also, this flag enables us to easily switch on and off this feature without worrying about introducing bugs while changing code.